### DISCUSSION TITLE
Reading a large .json from a website URL

### DISCUSSION THREAD

**Discussion by JLT032** [networkx/networkx#4747] [Q&A]
Dear,

There is a .json offered downloadable from a website, it weighs around 30MB. 

Thus far i've not found a way to load it from networkx, this appears undocumented.

Let alone get a grip on how to datamine it with networkx, which will involve exploring the datastructure, reading other .json from the same URL space with similar and uniformly structured information.

Please advise.

**Comment by rossbar** [comment]
NetworkX has a [json_graph](https://networkx.org/documentation/stable/reference/readwrite/json_graph.html) module in the `readwrite` package dedicated I/O w/ data in JSON format. Of course, there's nothing graph-specific about JSON itself, so whether or not one of the functions will work for you out-of-the-box depends on the structure of the data in the file you are trying to load.
